src/Avalonia.Controls/ProgressBar.cs

```diff
@@ -103,6 +115,18 @@ public double Container2AnimationEndPosition
                 get => _container2AnimationEndPosition;
                 set => SetAndRaise(Container2AnimationEndPositionProperty, ref _container2AnimationEndPosition, value);
             }
+
+            public double IndeterminateStartingOffset
```

Comment:
I had no idea how to comment this and the next property even when they were part of the main class. If you know what these properties do, is there any way to comment them?

Reply:
tbh i dont know what to put in the comments on those as well. When we ported them from WinUI they were unexplained. Right now we just want to clean this up bit by bit.

Reply:
Those properties aren't in WinUI. I think they were added just for Avalonia.

If you have a link to the Microsoft docs though I can comment them later.

Reply:
Basically they just adjust the animation start/end targets so that the indicator will show like it's looping around seamlessly on the simple theme progress bar

Reply:
Yea... theme-specific special properties sounds like a no-no

Reply:
Agreed